crates/dojo-lang/src/plugin.rs

```diff
@@ -316,6 +326,41 @@ fn get_additional_derive_attrs_for_model(derive_attr_names: &[String]) -> Vec<St
     additional_attrs
 }
 
+// parse the configuration file of the first crate to extract
+// the main package Id (so the name field of the package section of the Scarb.toml file)
+fn get_package_id(db: &dyn SyntaxGroup) -> Option<String> {
```

Comment:
Good that we've access to the info from the db. We should add a TODO in the comment of this function or a note to ensure that when we switch to the new macro system, we can ask scarb team to actually expose such information (as we will not have access to `db` anymore).